doc/tutorials/outline.md

```diff
@@ -0,0 +1,88 @@
+# Tutorial Outline
+
+These guides already exists
+
+- [components](components.md)
+- [development](development.md)
+- [interactivity](interactivity.md)
+- [layouts](layouts.md)
+- [param](param.md)
+- [structure](structure.md)
+- [styling](styling.md)
+
+They should be refactored into the below
+
+## Beginner
+
+These guides are for you that is just starting to discover Panel.
+
+You will learn to develop and deploy amazing tools and apps that can be contained within a single Python file or notebook.
+
+- [ ] Getting installed (just a link to getting started)
+  - [ ] Using a pre-installed environment for this tutorial: Purpose is to lower Barrier of entry. Make it possible to do at large conference with poor internet. Use Binder, Github spaces, Anaconda Notebooks ???
+- [ ] [Serve Apps](https://github.com/holoviz/panel/blob/docs_fixes_1.4_a1_review/doc/tutorials/beginner/panel_serve.md)
+- [ ] [Develop in an Editor](https://github.com/holoviz/panel/blob/docs_fixes_1.4_a1_review/doc/tutorials/intermediate/develop_editor.md)
+- [ ] [Develop in a notebook](https://github.com/holoviz/panel/blob/docs_fixes_1.4_a1_review/doc/tutorials/beginner/develop_notebook.md)
+- [ ] [Display with `pn.panel`](https://github.com/holoviz/panel/blob/docs_fixes_1.4_a1_review/doc/tutorials/beginner/display_pn_panel.md)
+- [ ] Display Content Efficiently:  panes - Panels output components
```

Comment:
Its compared to `pn.panel`. `pn.panel` comes with overhead of testing all Panes before it pick a specific pane to display with. It you want to write the most efficient (performant?) apps then you should select specific panes instead of using `pn.panel`.